main/perl-universal-require/APKBUILD

```diff
@@ -1,39 +1,39 @@
 # Contributor: Natanael Copa <ncopa@alpinelinux.org>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=perl-universal-require
-pkgver=0.13
+pkgver=0.18
 pkgrel=0
 pkgdesc="Require() modules from a variable"
-url="http://search.cpan.org/dist/UNIVERSAL-require/"
+url="https://search.cpan.org/dist/UNIVERSAL-require/"
 arch="noarch"
 license="GPL+ or Artistic"
 depends="perl"
 makedepends="perl-dev"
 install=""
 subpackages="$pkgname-doc"
-source="http://search.cpan.org/CPAN/authors/id/M/MS/MSCHWERN/UNIVERSAL-require-$pkgver.tar.gz"
+source="https://search.cpan.org/CPAN/authors/id/N/NE/NEILB/UNIVERSAL-require-$pkgver.tar.gz"
 
 _builddir="$srcdir"/UNIVERSAL-require-$pkgver
 prepare() {
```

Comment:
This prepare() function is not needed as default_prepare already apply any downstream patch